packages/angular_devkit/build_optimizer/src/transforms/import-tslib.ts

```diff
@@ -8,6 +8,12 @@
 import * as ts from 'typescript';
 
 
+export function testImportTslib(content: string) {
```

Comment:
Could this use the indexOf method with just the helper names?  Odds are if one of them shows up it's a helper definition.